frontend/site/.vitepress/config.js

```diff
@@ -0,0 +1,19 @@
+import { defineConfigWithTheme } from 'vitepress'
+
+// https://vitepress.dev/reference/site-config
+/**
+ * @typedef {{apiUrl: string}} OuvertureThemeConfig
+ */
+
+/** @type {import("vitepress").UserConfig<OuvertureThemeConfig>} */
+export default defineConfigWithTheme({
+  outDir: "../dist",
+  title: "Ouverture data.gouv.fr",
+  description: "Tableau de suivi des ouvertures de données, codes sources et API publics",
+  markdown: {
+    externalLinks: { target: "" },
+  },
+  themeConfig: {
+    apiUrl: process.env.VUE_APP_API_URL || "http://localhost:5000/api",
```

Comment:
If we want, we should configure VUE_APP_API_URL with env variables, right?

Reply:
Yes, when we do the vitepress build. I'll add it to the README.